cmd/operator-controller/main.go

```diff
@@ -407,10 +413,21 @@ func run() error {
 		crdupgradesafety.NewPreflight(aeClient.CustomResourceDefinitions()),
 	}
 
+	// determine if PreAuthorizer should be enabled based on feature gate
+	var preAuth authorization.PreAuthorizer
+	if features.OperatorControllerFeatureGate.Enabled(features.PreflightPermissions) {
+		setupLog.Info("preflight permissions check enabled via feature gate")
+		preAuth = authorization.NewRBACPreAuthorizer(mgr.GetClient())
+	} else {
+		setupLog.Info("preflight permissions check disabled via feature gate")
```

Comment:
Nit: Rather than one-off logging like this, it's probably better if we have a general "log the feature gate status" function that does this kind of thing consistently and in one place.

Reply:
done in 95145c63